docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
```

Comment:
I don't really agree with that. I think it takes focus where it's not needed. But one thing I think is important to add is a check that the property is provided, or else it will throw an exception.